### DISCUSSION TITLE
Web3Auth: Custom Auth Verifier URL Modification

### DISCUSSION THREAD

**Discussion by kevinrodriguez-io** [Web3Auth/Web3Auth#1023] [Help]
Is it possible to change the JWKs callback url location (Assuming is exactly the same JWKs)?
Or if I recreate the verifier with a same name and a different verifier will my users still be able to access to their existing wallets?

**Comment by shahbaz17** [comment]
Hey @kevinrodriguez-io 

For Testnet verifiers, you would be able to update it. But not for `mainnet`, `cyan`, `aqua` or `celeste`.
